docs/guide-explore.md

```diff
@@ -257,3 +257,89 @@ You now get a table representing the amount of samples in both biobanks per type
 Finally, select "patientid" in the third dropdown, the distinct attribute.
 The table will update to show you how many patients with at least one sample of a specific type are available in a biobank.
 
+## Data Explorer v2 (Experimental)
+
+The data explorer can be used to explorer the data
+
+#### Enabling the data explore v2 
+The data explorer v2 can be set as the default data explorer by removing the data-explorer (v1) plugin from the menu.
+Selecting an item in the navigator will then open the item in the data explorer v2
+
+The data explorer v2 can be added to the (main)menu via a redirect plugin 
+- Open the menu-manager
+- In de section 'Create a new menu item'
+  - Select ```redirect``` as plugin type
+  - Choose a name ( for example: 'Dataexplorer')
+  - Paste a link to app as the queryString
+```url=%2F@molgenis-ui%2Fdata-explorer%2Fdist%2Findex.html```
+  - Add the item to the menu 
+	
+### Data explore v2 features
```

Comment:
missing 'r' in data explorer

Reply:
tnks , please feel free to fix typo's inline during the review